asf-site-src/config.rb

```diff
@@ -119,24 +119,17 @@ def releasenotes(output, version)
 end
 
 def build_release_docs(output, version)
-  # TODO: get the version date from jira and do an up to date check instead of building each time.
   puts "Building docs for release #{version}"
-  puts "\tcleaning up output directories in #{output}"
-  FileUtils.rm_rf("#{output}/build-#{version}", secure: true)
-  FileUtils.rm_rf("#{output}/#{version}", secure: true)
 
   puts "Downloading and extracting #{version} from ASF archives"
   `(cd #{output} \
-    && mkdir -p build-#{version} \
+    && mkdir -p site/documentation/#{version} \
     && curl --fail --location --output site-#{version}.tar.gz \
     https://archive.apache.org/dist/yetus/#{version}/apache-yetus-#{version}-site.tar.gz \
-    && tar -C build-#{version} \
-    --strip-components 3 -xzpf site-#{version}.tar.gz \
+    && tar -C site/documentation/#{version} \
+    --strip-components 3 -xzkpf site-#{version}.tar.gz \
```

Comment:
You want to keep old files? What old files would you expect to be found under this path?

Reply:
See below.